metadata.go

```diff
@@ -38,12 +38,20 @@ const (
 	// in nanoseconds.
 	MetadataReadAt = cpluginv1.MetadataReadAt
 
-	// MetadataConduitPluginName is a Record.Metadata key for the name of the
-	// plugin that created this record.
-	MetadataConduitPluginName = cpluginv1.MetadataConduitPluginName
-	// MetadataConduitPluginVersion is a Record.Metadata key for the version of
-	// the plugin that created this record.
-	MetadataConduitPluginVersion = cpluginv1.MetadataConduitPluginVersion
+	// MetadataConduitSourcePluginName is a Record.Metadata key for the name of
+	// the source plugin that created this record.
+	MetadataConduitSourcePluginName = cpluginv1.MetadataConduitSourcePluginName
+	// MetadataConduitSourcePluginVersion is a Record.Metadata key for the
+	// version of the source plugin that created this record.
+	MetadataConduitSourcePluginVersion = cpluginv1.MetadataConduitSourcePluginVersion
+	// MetadataConduitDestinationPluginName is a Record.Metadata key for the
+	// name of the destination plugin that has written this record
+	// (only available in records once they are written by a destination).
+	MetadataConduitDestinationPluginName = cpluginv1.MetadataConduitDestinationPluginName
+	// MetadataConduitDestinationPluginVersion is a Record.Metadata key for the
+	// version of the destination plugin that has written this record
+	// (only available in records once they are written by a destination).
+	MetadataConduitDestinationPluginVersion = cpluginv1.MetadataConduitDestinationPluginVersion
```

Comment:
Do we have these documented somewhere?

Reply:
@lovromazgon is working on these changes

Reply:
They are documented here and in the [connector protocol](https://github.com/ConduitIO/conduit-connector-protocol/blob/main/cpluginv1/metadata.go) for now. But they are not used in any plugin yet afaik.